### ISSUE TITLE
Serversetup.sh script fails on windows

### ISSUE THREAD

**Issue by seanvree** [scytherswings/Plex-Board#77]
Issue:  Serversetup script fails on windows due to incorrect command.

Error:

```
c:\Plex-Board>sh serversetup.sh

The current operating system was detected as: Windows.
Stopping server if it's running.
serversetup.sh: line 26: source: stopServer.sh: file not found

c:\Plex-Board>sh stopserver.sh
It seems Puma isn't running right now. Nothing to kill.

c:\Plex-Board>sh serversetup.sh

The current operating system was detected as: Windows.
Stopping server if it's running.
serversetup.sh: line 26: source: stopServer.sh: file not found
```

Add "SH" on line26 of serversetup.sh  in front of "serversetup.sh"  for windows environment

Should look like this:

```
printf "\nStopping server if it's running.\n"
source sh stopServer.sh

```

**Comment by scytherswings** [comment]
Thanks @seanvree I'll retry this when I get back home. It's tough getting it right on all these different systems 😆

**Comment by scytherswings** [comment]
Closing this since it should be even more fixed by `dev` branch on v0.12.0
